Makefile

```diff
@@ -92,5 +92,14 @@ generate-api:
 vps:
 	VBoxManage list runningvms
 
+tt:
```

Comment:
I'm fine with this but let's give this a better name. `make tt` is not very descriptive.